registrations/utils.py

```diff
@@ -168,6 +168,15 @@ def move_waitlisted_to_attending(registration, count: int):
             registration.move_first_waitlisted_to_attending(first_on_list=first_on_list)
 
 
+def get_checkout_url_with_lang_param(checkout_url, lang_code):
+    if "?" in checkout_url:
+        checkout_url += f"&lang={lang_code}"
+    else:
+        checkout_url += f"?lang={lang_code}"
+
+    return checkout_url
```

Comment:
Currently mutates `checkout_url`, I think 🤔 
```suggestion
    if "?" in checkout_url:
        return f"{checkout_url}&lang={lang_code}"
    else:
        return f"{checkout_url}?lang={lang_code}"
```

Reply:
Apparently the result of `+=` is a new string, but can have the same memory address which is due to optimizations in CPython. But I still changed this as suggested to not leave anything based on assumptions.